lib/fictium/exporters/open_api/v3_exporter/content_formatter.rb

```diff
@@ -0,0 +1,20 @@
+module Fictium
+  module OpenApi
+    class V3Exporter
+      class ContentFormatter
+        def format(http_object, default = nil)
+          type = (http_object.presence && http_object[:content_type].presence) || default
```

Comment:
I'm not suggesting that you should do it, but you can do
`http_object&.[](:content_type).presence`

![image](https://user-images.githubusercontent.com/26874630/67907934-729b8780-fb58-11e9-86b9-ab3bdb0d2642.png)